nixos/modules/config/xdg/portals/termfilechooser.nix

```
@@ -0,0 +1,79 @@
+{ config
+, pkgs
+, lib
+, ...
+}:
+with lib; let
```

```suggestion
let
```

And use `lib.` explicitly.